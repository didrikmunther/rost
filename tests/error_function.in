let a1 = 1;
let a2 = 1;
let a3 = 1;
let a4 = 1;
let a5 = 1;
let a6 = 1;

fn b(b: int, c: int) {}

a(2, 5);
b(1, 10);