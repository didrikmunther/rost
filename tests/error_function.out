  --> ./tests/error_function.in:10:1 => CompilerError
 9 | 
10 | a(2, 5)
   | ^
   | └─ Undefined function: a
11 | b(1, 10)